lib/ap/internal/sys9.h

```diff
@@ -91,13 +89,13 @@ enum
 // Remove unused functions now from libc
 
 extern	int	await(char*, int);
-//extern	int	alarm(unsigned long);
+//extern	int	alarm(uint32_t);
 #ifndef __SYS_SOCKET_H__
 extern	int	bind(const char*, const char*, int);
 #endif
-extern	int	chdir(const char*);
+extern	int	_CHDIR(const char*);
```

Comment:
our chdir is compatible with linux and we should be using the same system call number.